src/bilder/images/edxapp/packer.pkr.hcl

```diff
@@ -66,30 +66,39 @@ source "amazon-ebs" "edx-platform" {
 
 build {
   sources = [
-    "source.amazon-ebs.edx-platform",
+    "source.amazon-ebs.edxapp",
   ]
 
   provisioner "shell-local" {
     inline = [
-      "echo '${build.SSHPrivateKey}' > /tmp/packer-session.pem",
-      "chmod 600 /tmp/packer-session.pem"
+      "echo '${build.SSHPrivateKey}' > /tmp/packer-${build.ID}.pem",
+      "chmod 600 /tmp/packer-${build.ID}.pem"
+    ]
+  }
+  provisioner "shell-local" {
+    environment_vars = [
+      "DEBIAN_FRONTEND=noninteractive"
+    ]
+    inline = [
+      "pyinfra --sudo --user ${build.User} --port ${build.Port} --key /tmp/packer-${build.ID}.pem ${build.Host} apt.packages packages='[\"git\", \"libmariadbclient-dev\", \"python3-pip\", \"python3-venv\", \"python3-dev\", \"build-essential\"]' upgrade=True update=True"
     ]
   }
   provisioner "shell" {
-    environment = [
-      "EDX_ANSIBLE_BRANCH=${var.ansible_branch}"
+    environment_vars = [
+      "EDX_ANSIBLE_BRANCH=${var.ansible_branch}",
     ]
     inline = [
-      "sudo apt-get update && sudo apt-get -y upgrade",
-      "sudo apt-get install -q -y git libmysqlclient-dev",
-      "sudo apt-get install -q -y python3-pip python3-venv python3-dev build-essential",
       "cd /tmp && git clone https://github.com/edx/configuration --depth 1 --branch $EDX_ANSIBLE_BRANCH",
       "cd /tmp/configuration && python3 -m venv .venv && . .venv/bin/activate && pip install -r requirements.txt"
     ]
   }
   provisioner "ansible-local" {
-    playbook_file = "files/edx_${var.node_type}_playbook.yml"
+    playbook_file = "${path.root}/files/edx_${var.node_type}_playbook.yml"
```

Comment:
Is this yml file available locally? Guess not sure I follow where this is coming from

Reply:
Yes, this is available in the repository. `path.root` is the same as `Path(__file__).parent` in Python.